integration-tests/baas-test-server/README.md

```diff
@@ -0,0 +1,21 @@
+# Run BaaS from source
+
+Run the Lerna bootstrap from the root of this repository.
+
+```shell
+npx lerna bootstrap --scope @realm/baas-test-server
+```
+
+Create a `.env` file with your `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` secrets.
```

Comment:
These should/could also be read from the local `env`

Reply:
Sure, that's possible ... Although you _really_ shouldn't expose secrets in your global environment as it's (very!) easy to accidentally leak those to a rouge package / app running on your system.